src/robusta/core/sinks/transformer.py

```diff
@@ -58,6 +58,10 @@ def apply_length_limit(msg: str, max_length: int, truncator: Optional[str] = Non
         truncator = truncator or "..."
         return msg[: max_length - len(truncator)] + truncator
 
+    @staticmethod
+    def apply_length_limit_to_markdown(msg: str, max_length: int, truncator: str = "...") -> str:
+        return trim_markdown(msg, max_length, truncator)
```

Comment:
i would put this in a try catch and if this fails try to run apply_length_limit

Reply:
done